src/Spryk/Model/Spryk/Filter/CamelCaseToDashFilter.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Filter;
+
+use Laminas\Filter\FilterChain;
+use Laminas\Filter\StringToLower;
+use Laminas\Filter\Word\CamelCaseToDash;
+
+/**
+ * Filter is used to convert a camelCased string
+ * into string where words are separated by "-".
+ *
+ * Example:
+ * $this->filter('stringExample') === `string-example';
+ */
+class CamelCaseToDashFilter implements FilterInterface
```

Comment:
We already have the same filter named `\SprykerSdk\Spryk\Model\Spryk\Filter\DasherizeFilter` please use the existing filter.